src/async.c

```diff
@@ -379,3 +381,23 @@ mpd_async_recv_line(struct mpd_async *async)
 
 	return src;
 }
+
+struct mpd_binary
+mpd_async_recv_binary(struct mpd_async *async, const unsigned binary)
+{
+	struct mpd_binary src;
```

Comment:
Why is this named `src`? This isn't a "source" of anything.

Reply:
function is completely rewritten